themes/kiali/layouts/partials/home/about-section.html

```diff
@@ -2,11 +2,12 @@
     <h1>About Kiali</h1>
 
     <p>
-    Kiali is an observability console for Istio with service mesh configuration
-    capabilities. It helps you to understand the structure of your service mesh
-    by inferring the topology, and also provides the health of your mesh. Kiali
-    provides detailed metrics, and a basic Grafana integration is available for
-    advanced queries. Distributed tracing is provided by integrating Jaeger.
+    Kiali is a management console for Istio-based service mesh.  It provides
+    dashboard, observability and configuration capabilities. It helps you to
+    understand the structure of your service mesh by inferring the topology and
+    also displays the health of your mesh. Kiali provides detailed metrics,
```

Comment:
I miss the verb "operate" the Service Mesh.
Some way that is clear that Kiali performs actions that involve complex configuration to allow manage Istio.

Reply:
I like the term "management console", let me then also work in the  "operate" verb:  How about:

> Kiali is a management console for Istio-based service mesh.  It provides
> dashboards, observability and lets you to operate your mesh with robust
> configuration and validation capabilities.  It shows the structure of your
> service mesh by inferring traffic topology and displays the health of your
> mesh.  Kiali provides detailed metrics, ...

Reply:
It sounds great to me.